### PR TITLE
S03 us09 crud user form connexion

### PR DESCRIPTION
**Pull request by plbonnier**

### CODE REVIEW

**Review by Thomas-Lunardo — APPROVED** [review]

File: src/Controller/RegistrationController.php [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+            // generate a signed url and email it to the user
+            $this->emailVerifier->sendEmailConfirmation(
+                'app_verify_email',
+                $user,
+                (new TemplatedEmail())
+                    ->from(new Address('p3.nomnomnom@gmail.com', 'NomNomNom'))
+                    ->to($user->getEmail())
+                    ->subject('Veuillez confirmer votre mail')
+                    // ->htmlTemplate('registration/confirmation_email.html.twig')
```
Comment (StoicFrenchDev) [inline src/Controller/RegistrationController.php:58]: Happy New Year

**Review by StoicFrenchDev — CHANGES_REQUESTED** [review]

File: src/Entity/User.php [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+        return $this;
+    }
+
+    /**
+     * @see UserInterface
+     */
+    public function eraseCredentials(): void
+    {
+        // If you store any temporary, sensitive data on the user, clear it here
+        // $this->plainPassword = null;
```
Comment (StoicFrenchDev) [inline src/Entity/User.php:115]: is this supposed to be here

**Review by StoicFrenchDev — CHANGES_REQUESTED** [review]

File: templates/registration/index.html.twig [hunk 3]
```diff
@@ -0,0 +1,20 @@
+{% extends 'base.html.twig' %}
+
+{% block title %}Hello RegistrationController!{% endblock %}
+
+{% block body %}
+<style>
```
Comment (StoicFrenchDev) [inline templates/registration/index.html.twig:6]: Heresy!

**Review by StoicFrenchDev — CHANGES_REQUESTED** [review]

File: templates/registration/index.html.twig (hunk 3 above)
Comment (StoicFrenchDev) [reply templates/registration/index.html.twig:6]: Remove the style tag

**Review by StoicFrenchDev — CHANGES_REQUESTED** [review]

**Review by StoicFrenchDev — APPROVED** [review]

**Pull request merged**